src/Job/NormalJob.ts

```diff
@@ -8,13 +18,16 @@ export class NormalJob implements Job {
 
 	/**
 	 * @param name - The job's name. Must be unique between jobs in the same dependency graph.
-	 * @param prerequisites - An optional set containing all of this job's prerequisites. Defaults to no prerequisites.
+	 * @param prerequisites - A set containing all of this job's prerequisites. Defaults to no prerequisites.
+	 * @param environment - A description of the job's runtime environment. For forwards compatibility, always provide this parameter. Defaults to a Docker image containing GNU Make.
```

Comment:
```suggestion
	 * @param environment - A description of the job's runtime environment. For forwards compatibility, always provide this parameter. Defaults to the name of a Docker image containing GNU Make.
```

Reply:
I want to keep this language generic (just like I made a `JobEnv` interface as opposed to a string containing the image name). There may be other details aside from the image name that we need in the future.